testsuite/tests/kuadrant/reconciliation/cr_configuration/test_reconcile_limitador.py

```diff
@@ -0,0 +1,64 @@
+"""Tests for Kuadrant sub component - Limitador CR configuration via Kuadrant CR"""
+
+import pytest
+
+pytestmark = [pytest.mark.kuadrant_only]
+
+
+@pytest.fixture()
+def commit():
+    """Omitting authorization and rate limiting as it is not needed"""
+
+
+@pytest.fixture(autouse=True)
+def kuadrant_clean_up(request, kuadrant):
+    """
+    Return fields to default values.
+    This can be simplified once https://github.com/Kuadrant/kuadrant-operator/issues/617 is fixed.
+    """
+
+    def _finalize():
+        kuadrant.limitador = {"replicas": 1, "resourceRequirements": {"requests": {"cpu": "250m", "memory": "32Mi"}}}
```

Comment:
I'd better set those values at the beginning of the tests in case the finalizer is not executed, but I see it will be irrelevant when this issue is resolved

Reply:
In both tests I am changing values in "setup" state. So this finalizer is really just for cleanup